### PR TITLE
feat(ui): put observation tables first

### PR DESCRIPTION
**Pull request by martinmaillard**

### CODE REVIEW

File: ui/src/components/CsvSourceMapping.vue [hunk 1]
```diff
@@ -112,7 +112,9 @@ export default class CsvSourceMapping extends Vue {
   selectedColumns: CsvColumn[] = []
 
   get sourceTables (): Table[] {
-    return this.tables.filter(({ csvSource }) => csvSource?.id.equals(this.source.id))
+    return this.tables
+      .filter(({ csvSource }) => csvSource?.id.equals(this.source.id))
+      .sort(({ isObservationTable: o1 }, { isObservationTable: o2 }) => (o1 === o2) ? 0 : (o1 ? -1 : 1))
```
Comment (lucafurrer) [inline ui/src/components/CsvSourceMapping.vue:117]: Just wondering, should we sort the rest alphabetically or something?

**Review by lucafurrer — APPROVED** [review]

File: ui/src/components/CsvSourceMapping.vue (hunk 1 above)
Comment (tpluscode) [reply ui/src/components/CsvSourceMapping.vue:117]: Or in the exact order they were created, with option to change that :)

**Review by tpluscode — APPROVED** [review]

File: ui/src/components/CsvSourceMapping.vue (hunk 1 above)
Comment (martinmaillard) [reply ui/src/components/CsvSourceMapping.vue:117]: That would be great. But the really important thing visually is that observation tables appear first.

Comment (tpluscode) [reply ui/src/components/CsvSourceMapping.vue:117]: image

**Pull request merged**